src/lib/pages/evm-tx-details/components/evm-tx-method/EvmTxTransferErc20.tsx

```diff
@@ -50,20 +48,14 @@ export const EvmTxTransferErc20 = ({
             textVariant="body1"
             ampCopierSection="tx_page_message_header_send_address"
           />{" "}
-          <EvmMethodChip txInput={input} width="110px" />{" "}
-          {formatTokenWithValue(amount)} to{" "}
-          {toAddress?.address ? (
-            <ExplorerLink
-              type={toAddress.type}
-              value={toAddress.address}
-              showCopyOnHover
-              textVariant="body1"
-            />
-          ) : (
-            <Text variant="body2" color="text.disabled">
-              -
-            </Text>
-          )}
+          <EvmMethodChip txInput={input} txTo={erc20Contract} width="110px" />{" "}
+          {formatTokenWithValue(amountToken)} to{" "}
+          <ExplorerLink
+            type="user_address"
```

Comment:
What if the `to_address` is a contract address?

Reply:
we never handle at case in the first place cause we cannot detect the different between `user_address` and `contract_address` for the transfer

Reply:
I think we can do [this endpoint](https://lcd.minievm-2.initia.xyz/minievm/evm/v1/codes/0xeA5d859BeCfFA4f4bc12ee96cAF85EFc456d54D9). If it returns "0x", it’s a user address; otherwise, it’s a contract address. We could implement this on the address details page to auto-redirect if it’s a contract. We can do this in the next PR.

Reply:
we already have this logic in address details page